### DISCUSSION TITLE
[Feature request] Multilingual Parallel Comparison Support

### DISCUSSION THREAD

**Discussion by ryanlo713** [vuepress/core#1362] [Ideas]
### Clear and concise description of the problem

I kindly request the VuePress project to implement a new feature that enables multilingual parallel comparison, similar to the documentation style used by W3C (World Wide Web Consortium) documents, as seen in the provided screenshots.

### Requested Functionality:

The desired features for the multilingual parallel comparison support are as follows:

* Multilingual support for titles and paragraphs, allowing for side-by-side comparison of content in different languages.
* The ability to toggle the visibility of content in specific languages by clicking on adjacent buttons.

### Screenshots:

Please refer to the attached screenshots that demonstrate the desired multilingual parallel comparison feature.

![image](https://github.com/vuepress/vuepress-next/assets/12672523/9f08bbaa-8cff-43be-9855-b3e89abaf835)

When clicking on the "English" button:

![image](https://github.com/vuepress/vuepress-next/assets/12672523/74115f01-5948-4014-9bd9-c572e8781985)

Thank you for considering this feature request. I believe it would greatly enhance the usability and accessibility of the project.

### Additional Information:

Please let me know if you require any further details or information to proceed with this feature request.

### References

* [Requirements for Chinese Text Layout](https://www.w3.org/TR/clreq/)
* [Requirements for Japanese Text Layout](https://www.w3.org/TR/jlreq/)

**Comment by Mister-Hope** [comment]
I don't think we are interest in this.

- Features like this could be a new theme, we have thene API so anyone can achieve this
- This is not what default theme should support, it breaks everything and is not friendly to docs with only 1 language.

So this question is out of scope of this repo, closing it and moving it to dicussion.
